### PR TITLE
FZ83P-6

### PR DESCRIPTION
**Pull request by bartzaw**
Hi folks, please check 1st version of featues section. Let me know if anything.

### CODE REVIEW

**Review by Decomplosed — APPROVED** [review]
seems fine to me

File: index.html [hunk 1]
```diff
@@ -42,34 +42,34 @@ <h1>Creating a unique look<br><span>Never been easier.</span></h1>
         <section class="section" id="features">
             <div class="features-items">
                 <h1>Poszerz swoje możliwości</h1>
-                <span>
```
Comment (icelandico) [inline index.html]: Maybe it's better to nest `<span>` within `<div>` , not backwards?

File: assets/style/style.css [hunk 2]
```diff
@@ -1,6 +1,44 @@
...
+}
+
+.features-items-description {
+    position: relative;
+    width: 100%;
+    text-align: center;
+
+}
+
+#add-info {
```
Comment (icelandico) [inline assets/style/style.css]: Following the suggestion of @sebastian-kreft on my request - We shall not use #id as a selector.

**Review by icelandico — CHANGES_REQUESTED** [review]
We should take account of @sebastian-kreft suggestions :)

**Comment by bartzaw** [comment]
I removed those IDs as changes I've put in there were completely redundant.

**Review by icelandico — APPROVED** [review]
Great!

**Pull request merged**